core/trino-main/src/test/java/io/trino/operator/TestFlatHashStrategy.java

```diff
@@ -51,14 +51,12 @@
 
 public class TestFlatHashStrategy
 {
-    private static final TypeOperators TYPE_OPERATORS = new TypeOperators();
-    private static final JoinCompiler JOIN_COMPILER = new JoinCompiler(TYPE_OPERATORS);
-
     @Test
     public void testBatchedRawHashesZeroLength()
     {
-        List<Type> types = createTestingTypes();
-        FlatHashStrategy flatHashStrategy = JOIN_COMPILER.getFlatHashStrategy(types);
+        TypeOperators typeOperators = new TypeOperators();
```

Comment:
Is this point with moving these, to reduce testing memory or is there another goal.  If it just testing memory, you can make the fields instant fields, as Junit5 dereferences test classes properly (unlike TestNG)

Reply:
Yep, will make these instance fields.